config/config.dev.enc.yaml

```diff
@@ -1,66 +1,76 @@
+            client_id: ENC[AES256_GCM,data:FacaKTXpofXMtPfSu7aSnwFkt9sE6YU=,iv:O1qlInVUABDSkykut6oZWjeKTAoF9Fo8rDcAqztGgrQ=,tag:4pZTf5dAyrjklCVwpe7hug==,type:str]
+            client_secret: ENC[AES256_GCM,data:GYdU9PGGNsCbea8E6QGkdYG/QXALHpGW+HZqhf0Xh6o=,iv:hBR8WRvaJ1o/Nim39OLNkZLmr59LBWshLx0a3Dxe+H0=,tag:+H4/MMuoWYn5QPOI1tshFA==,type:str]
 mqtt:
-    broker: ENC[AES256_GCM,data:4r9dkIPitE5OtbjxSj63OuudNugheFf6/g6F0L83SF7ymCE=,iv:uvX29U6A4JZ0uWtc+WEpBzI3t7AbqemRR+zpI7EJGR4=,tag:/P0jK9rI7PbsBNQXQi/40Q==,type:str]
-    client_id: ENC[AES256_GCM,data:o8tCZJ4FHscHKIs=,iv:5ie4bA22mO+cz3pJUQPxq8NEAqNQI6cGIjdMCt62nmY=,tag:8uwqtG/IpUFbYS/odgxU/w==,type:str]
-    username: ENC[AES256_GCM,data:R6K7UVHzImx33vwpBxlsHw==,iv:6yhHggVpJlQ0Tv8RZwUi5JSsiM8uGW1sw1gdEACi2t4=,tag:8wKXOdRbp3VKKss7FNJTdQ==,type:str]
-    password: ENC[AES256_GCM,data:gG0qLFy1hjNrBdMbHUxrkjrE0ZMJ5cHO2zULeTIaCDK7tuvHggmfBTKgv2tSgnoHXt2m/dqD0XCm7LMMbR2lSdnzb3KZ0aCBe7KMfQ78hir1qgUtidwrPuHw0cNzvuDtFgs=,iv:ZU98B5W5t9NHr8rf9EdGm5dqIKXCmRdqQI6noVNr4wI=,tag:fX0ms+qmlt1CMXcibN+72g==,type:str]
-    topic: ENC[AES256_GCM,data:CYAGFXrjSZHdK2lZ9fpkrL1fWZH8eG9PhkBDScX6tlfq4kSMHD3qGUWJ,iv:q0L/iCO2PmkMLUedUVBaTd6uTqpzTACXDn9KuvinXuc=,tag:3CYm2Iievm/SIFGOHjcpag==,type:str]
+    broker: ENC[AES256_GCM,data:pT8xpJEW0sxuOkoPPiJTe3r2NuOpprseoqkvA7elmZgnwU8=,iv:jw/5i5sI7DL6yZqlonTaiZvFEs0vhcS3JYZoJxf9Ers=,tag:Uf4hpl7w+pQJCXxkQxSUhg==,type:str]
+    client_id: ENC[AES256_GCM,data:uR3CtqlqNNVA8GI=,iv:svVpOXwgvZg4hAYgDZw//KoH0rDQ8/ohVfAayyo7Iko=,tag:okFFq7JcHv3VB2/UbGYzCg==,type:str]
+    username: ENC[AES256_GCM,data:vMCjn6aBpkYwEmAUFY2bHA==,iv:z5/bOAm2H/qjzIyRY+CadKQDToA6WXICBWuKPzSAtkE=,tag:/73Hk6Eiyz3w+COK/b82GA==,type:str]
+    password: ENC[AES256_GCM,data:jLfRp4fEYh/JSj07krL+DzfLMQwlMjJWz6fcYxXHnawVIXUySEnN55oBzgB+hy9kIBzjUG5r2QCtuKao3osqXSeG0TLdw0p2hRZxu2Lv48OjzbKgStfoEuluF9Jbvh6MCww=,iv:byEkHjc9q7qhIakh6T+405wK2bRKolt51OxEmVl3GAE=,tag:BPzumB+LQukTwUWpZy5n3w==,type:str]
+    topic: ENC[AES256_GCM,data:e9wj1qn2IANYc5de8cpOWa4IS1NE16g2NkD8Rk1UHmk8egU/45QtU7C1,iv:UCs92zl2k7gtdELhsGtjGo2GDH6K3kXoUVKT0SNtu7U=,tag:YvmvMFCw3W5wXTUUu223MA==,type:str]
+map:
+    center:
+        - ENC[AES256_GCM,data:OdVy0R4CWPCFKFNJojpPUT4J,iv:pD5UlCN8AqzFQwV4p7e/ZuLPNTQQkKXEqPuoWXeYwvw=,tag:pfzK0d1l0nPnSJFJ9KQ2Hw==,type:float]
+        - ENC[AES256_GCM,data:FZ7ylEiIAnwqaTErDrBduw==,iv:9DgaUtV5Q/I8PL2frWUAYX5PaXz5aJm76deUUDx6GUs=,tag:cZ9Cfj38r68vSyLTfq9seg==,type:float]
+    bound_south_west:
+        - ENC[AES256_GCM,data:RZdkgHI=,iv:Jt+aNxVgjye0xdFmv/9aUvz3vgk0cGEmNGHO9HwBKUo=,tag:FyB463IfojTUpHkyZxb98A==,type:float]
+        - ENC[AES256_GCM,data:9mSL,iv:i/0ou+KWUuZNE5HzJ/8b+YHeRrZwit24Nxi84Zdjs8A=,tag:0fZiO9k5GD6Yn4G8bVZz5g==,type:float]
+    bound_north_east:
+        - ENC[AES256_GCM,data:fgsfuoA=,iv:Ls4AbZqa+lMWWBHZXzOnD+2FhHuBS43jkS6wcdEAeUU=,tag:/0upukfGhxAgKF5Y5p1ePA==,type:float]
+        - ENC[AES256_GCM,data:ybH3,iv:EmXoewDQQ2cWr33a80f7Vw6MROQAUwqxGuzFH2XtUKw=,tag:OUMb1BfbC3L4f73m+bbpwQ==,type:float]
```

Comment:
Bounding boxes are standardised and normally an array of 4 elements. So instead of specific specifications like `bound_south_west` or `bound_north_east` I would consider something like this: 

```yaml
map:
    center: [54.792277136221905, 9.43580607453268]
    bbox: [54.85, 9.6, 54.73, 9.3]
```

![image](https://github.com/user-attachments/assets/537d5ca1-b9c9-42f9-a6c4-7aacd9cca1de)
https://wiki.openstreetmap.org/wiki/Bounding_box